wow/sql/address_nychastats.sql

```diff
@@ -0,0 +1,29 @@
+WITH DEVS AS (
+	SELECT 
+		DISTINCT DEVELOPMENT 
+	FROM NYCHA_BBLS_18
+	WHERE DEVELOPMENT !~ 'POLICE SERVICE AREA'
+	AND BBL = %(bbl)s
+),
+
+DEV_BBLS AS (
+	SELECT DISTINCT ON (BBL)
+		N.DEVELOPMENT,
+		N.BBL,
+		P.UNITSRES, 
+		E.EVICTIONS
+	FROM NYCHA_BBLS_18 N
+	LEFT JOIN PLUTO_19V2 P USING(BBL)
+	LEFT JOIN (
+		SELECT BBL, COUNT(*) EVICTIONS FROM MARSHAL_EVICTIONS_ALL 
+		WHERE RESIDENTIALCOMMERCIALIND = 'RESIDENTIAL'
+		GROUP BY BBL
+	) E USING(BBL)
```

Comment:
What does this E mean?

Reply:
So `E` is the name I'm assigning to the subquery inside the preceding parentheses (short for "Evictions"). I tried to keep it consistent with the alias names I was assigning the other tables in the query (`N` and `P`):
```
FROM NYCHA_BBLS_18 N
LEFT JOIN PLUTO_19V2 P...
```